debian/not-installed

```diff
@@ -0,0 +1,3 @@
+usr/lib/x86_64-linux-gnu/liboidc-agent.a
```

Comment:
liboidc-agent.a should be installed from the lib-dev package

Reply:
But I had to remove it by request of the debdeveloper.